mkdocs.yml

```diff
@@ -0,0 +1,81 @@
+site_name: TerriaJS
```

Comment:
I'd prefer to keep this file in `doc/`. What's the reason to pull it out to the root?

Reply:
I'm not sure if this is due to a version change in mkdocs or something

If I keep `doc/mkdocs.yml` and set it to 
````
....
docs_dir: /
site_dir: ./../wwwroot/doc/guide
...
````

If I try and run `mkdocs build` I get something like 
````
ERROR   -  Config value: 'site_dir'. Error: The 'site_dir' should not be within the 'docs_dir' as this leads to the build directory being copied into itself and duplicate nested files in the 'site_dir'.(site_dir: '/Users/win167/Documents/Code/TerriaMapMobx/packages/terriajs/wwwroot/doc/guide', docs_dir: '/')
`````

Short answer is I can't make it work 🤷‍♂️

Reply:
Ahh ok. Actually I should probably merge my mkdocs update on `master` into this PR before we continue. I think it solves that issue.